### PR TITLE
Add Nyakwarnyanya's file

### PR DESCRIPTION
**Pull request by AgalTheAnalyst**
Created first Pull request.

### CODE REVIEW

**Review by AgalTheAnalyst — COMMENTED** [review]
Learn to respond to a Review

File: _posts/0000-01-02-Nyakwarnyanya.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: slide
+
+title: "Welcome to our second slide."
+
```
Comment (Nyakwarnyanya) [inline _posts/0000-01-02-Nyakwarnyanya.md:5]: Trying something new

Comment (Nyakwarnyanya) [inline _posts/0000-01-02-Nyakwarnyanya.md:5]: Pasaka yenye Baraka.

**Review by AgalTheAnalyst — COMMENTED** [review]
Why is this becoming hard?

**Pull request merged**